src/main/java/com/linecorp/armeria/server/http/tomcat/TomcatService.java

```diff
@@ -73,6 +76,10 @@ public static TomcatService forFileSystem(Path docBase) {
         return TomcatServiceBuilder.forFileSystem(docBase).build();
     }
 
+    public static HttpService forConnector(String hostname, Connector connector) {
```

Comment:
Exposing `connector()` looks a good idea. if so, why do we still expose `config()` as `connector()` will provide many tomcat variables?